website/content/en/preview/getting-started/migrating-from-cas/scripts/step08-generate-chart.sh

```diff
@@ -0,0 +1,7 @@
+helm template --namespace karpenter \
+    karpenter karpenter/karpenter \
+    --set aws.defaultInstanceProfile=KarpenterInstanceNodeRole \
+    --set clusterEndpoint="${OIDC_ENDPOINT}" \
+    --set clusterName=${CLUSTER_NAME} \
+    --set serviceAccount.annotations."eks\.amazonaws\.com/role-arn"="arn:aws:iam::${ACCOUNT_NAME}:role/KarpenterController" \
```

Comment:
can you use `AWS_ACCOUNT_ID` instead of `ACCOUNT_NAME` to match the other Getting Started Guides ?